public/app/features/migrate-to-cloud/onprem/ResourceDetailsModal.tsx

```diff
@@ -9,11 +9,36 @@ interface ResourceDetailsModalProps {
   onClose: () => void;
 }
 
+const resourceErrorMessages: { [key: string]: string } = {
+  DATASOURCE_NAME_CONFLICT:
+    'There is a data source with the same name in the target instance. Rename one of them and try again.',
```

Comment:
It would be good to make these translatable by using `t` instead of being hardcoded